build/SharedFxInstaller.targets

```diff
@@ -23,13 +23,15 @@
   <Target Name="_DownloadInstallers">
     <!-- Download dotnet installers -->
     <MakeDir Directories="$(_InstallerSourceDir)" />
-    <KoreBuild.Tasks.DownloadFile
-      Uri="$(RuntimeArchiveLinkPrefix)-%(WindowsSharedFxRIDs.Identity).zip$(DotNetAssetRootAccessTokenSuffix)"
-      DestinationPath="$(_InstallerSourceDir)$(DotnetRuntimeFileNamePrefix)-%(WindowsSharedFxRIDs.Identity).zip"
+    <DownloadFile
```

Comment:
@ryanbrandenburg this look correct to you?

Reply:
Yeah that looks correct.